netbox/extras/forms/filtersets.py

```diff
@@ -137,6 +138,22 @@ class ExportTemplateFilterForm(SavedFiltersMixin, FilterForm):
     )
 
 
+class ImageAttachmentFilterForm(SavedFiltersMixin, FilterForm):
+    fieldsets = (
+        (None, ('q', 'filter_id')),
+        ('Attributes', ('content_type', 'name',)),
```

Comment:
This needs to match the exact filter name, or it raises an exception.

```suggestion
        ('Attributes', ('content_type_id', 'name',)),
```